students/AlyssaHong/lesson03/mailroom.py

```diff
@@ -0,0 +1,66 @@
+list_col = ['Donor Name','Total Given','Num Gifts','Average Gift']
+list_donor_name = ['Fred','Alex','Henry','Alyssa','Leo']
+list_total_given = [7000, 30000, 5000, 120000, 107000]
+list_num_gifts = [2, 3, 1, 3, 2]
+list_avg_gift = [4500, 10000, 5000, 40000, 53500]
+
+def send_thanks():
+    user_input = input("Do you want to see the list of donor? Please type 'list'--> ")
+    if user_input == "list":
+        print('{:^15}'.format("==Donor Name=="))
+        for donor_name in list_donor_name:
+            print('{:^15}'.format(donor_name))
+        donor_name = input("Type the donor's name: ")
+        new_gift = int(input("Type your donations: "))
+
+        if donor_name in list_donor_name:
+            i = list_donor_name.index(donor_name)
+            print(donor_name, list_donor_name[i])
+            if list_donor_name[i] in list_donor_name:
+                list_total_given[i] = list_total_given[i] + new_gift
+                print(list_total_given[i])
+                list_num_gifts[i]  = list_num_gifts[i] + 1
+                print(list_num_gifts[i])
+                list_avg_gift[i] = list_total_given[i]/list_num_gifts[i]
+        else:
```

Comment:
line 25 and line 16 should be de-dented; user should be presented to either choose 'list' or add/update donor. Right now you are always forced to type `list`